test/controllers/admin/partners_controller_test.rb

```diff
@@ -49,7 +49,7 @@ class Admin::PartnersControllerTest < ActionDispatch::IntegrationTest
   it_allows_access_to_index_for(%i[neighbourhood_admin]) do
     get admin_partners_url
     assert_response :success
-    #assert_select 'a', 'Edit'
+    # assert_select 'a', 'Edit'
```

Comment:
Are the changes in this file just linting?

Reply:
Yeah pretty much lol